core/src/core/block.rs

```diff
@@ -744,29 +734,14 @@ impl Block {
 		Ok(kernel_sum)
 	}
 
-	// Verify the tx is not too big in terms of
-	// number of inputs|outputs|kernels.
+	// Verify the block is not too big in terms of number of inputs|outputs|kernels.
 	fn verify_size(&self) -> Result<(), Error> {
```

Comment:
I think we should call this `verify_weight()` if we're going to do this based on weight.
"size" felt really ambiguous to me when I added it (size of what?)
"weight" I think is specific enough to be documented clearly and reasonably intuitive.